src/contracts/libraries/AccountingMath.sol

```diff
@@ -3,10 +3,10 @@ pragma solidity 0.8.25;
 
 library AccountingMath {
     // Gas Accounting public constants
-    uint256 private constant _ATLAS_SURCHARGE_RATE = 1_000_000; // 1_000_000 / 10_000_000 = 10%
-    uint256 private constant _BUNDLER_SURCHARGE_RATE = 1_000_000; // 1_000_000 / 10_000_000 = 10%
-    uint256 private constant _SURCHARGE_SCALE = 10_000_000; // 10_000_000 / 10_000_000 = 100%
-    uint256 private constant _FIXED_GAS_OFFSET = 100_000;
+    uint256 public constant _ATLAS_SURCHARGE_RATE = 1_000_000; // 1_000_000 / 10_000_000 = 10%
+    uint256 public constant _BUNDLER_SURCHARGE_RATE = 1_000_000; // 1_000_000 / 10_000_000 = 10%
+    uint256 public constant _SURCHARGE_SCALE = 10_000_000; // 10_000_000 / 10_000_000 = 100%
+    uint256 public constant _FIXED_GAS_OFFSET = 100_000;
```

Comment:
These should be internal I think. If public, they should not have the `_` prefix in the name.